src/components/BrewForm.tsx

```diff
@@ -21,49 +24,51 @@ const useFirebaseUser = () => {
     return auth.currentUser;
 };
 
-const BrewForm = ({brewer}: { brewer: Brewer }) => {
-    const ratios = [
-        '1:1',
-        '1:2',
-        '1:3',
-        '1:4',
-        '1:5',
-        '1:6',
-        '1:7',
-        '1:8',
-        '1:9',
-        '1:10',
-        '1:11',
-        '1:12',
-        '1:13',
-        '1:14',
-        '1:15',
-        '1:16',
-        '1:17',
-        '1:18',
-        '1:19',
-        '1:20',
-    ];
+const BrewForm = ({ brewer }: { brewer: Brewer }) => {
+
+    const { multiple, dose, name: vehicle, img: brewerImg } = brewer;
+    const currentUser = useFirebaseUser();
 
     const [formFields, setFormFields] = useState<FormFields>({
         coffee: '',
+        multiple: multiple,
+        dose: dose,
         roast: '',
         grind: '',
-        rating: 5,
+        rating: 0,
     });
 
-    const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-        setFormFields({
-            ...formFields,
-            [e.target.name]: e.target.value,
+    const setFormFieldsHandler = (inputName: string, updatedValue: any) => {
```

Comment:
@SaboyaDev Something you can do here (for future reference) is have `string | number` or something of the sort. TS supports multiple types.